packages/artifact/src/config-variables.ts

```diff
@@ -0,0 +1,35 @@
+export function getUploadFileConcurrency(): number {
+  return 2
+}
+
+export function getUploadChunkConcurrency(): number {
```

Comment:
Why do we need both file concurrency and chunk concurrency?  So potentially we could open 6 connections for one job?  Seems a little bit aggressive.  Is this what we had in v1?

Reply:
Currently in the runner it's set to 2 concurrent file uploads and then each file is uploaded in 4MB chunks. As far as I can tell, chunks are not uploaded concurrently so there is a bump from 2 -> 6 connections: https://github.com/actions/runner/blob/6c70d53eead402ba5d53676d6ed649a04e219c9b/src/Runner.Plugins/Artifact/FileContainerServer.cs#L211

https://github.com/actions/runner/blob/6c70d53eead402ba5d53676d6ed649a04e219c9b/src/Runner.Plugins/Artifact/FileContainerServer.cs#L472

Perhaps the chunk upload concurrency can be bumped down to 2 instead of 3?

Reply:
If we bump this up, we will stress the server more - if there are no requests yet, we should keep the load the same.  This will hit our pipelines AT so we need to be careful.  I would say change the chunk concurrency at 1 for now.

Reply:
I'll change it to 1. The concurrency is already setup so in the future it will be easy to bump up if we ever want to play around with increasing it